koans/AssertKoans.php

```diff
@@ -7,67 +7,73 @@
      */
-    public function testAssertTruth()
+    public function checksThatTheAssertConditionIsTrue()
     {
-        // Change "false" to "true"
         $this->assertTrue(false);
     }
 
     /**
-     * @testdox Enlightenment may be more easily achieved with appropriate messages.
+     * @test
+     * @testdox Enlightenment may be more easily achieved with appropriate messages
      */
-    public function testAssertWithMessage()
+    public function checksIfANonEmptyArrayIsFalse()
     {
-        // Change "false" to "true"
         $this->assertTrue(false, "This should be True, please fix this");
     }
 
     /**
-     * @testdox Sometimes we will ask you to fill in the values.
+     * @test
+     * @testdox You can use assertEquals to check that two things are equal
      */
-    public function testFillInValues()
+    public function checksHowMuchIsOnePlusOne()
     {
-        // Replace __ with your answer
-        $this->assertEquals(__, 1+1);
+        $this->assertEquals(__, 1 + 1);
     }
 
     /**
-     * @testdox To understand reality, we must compare our expectations against reality.
+     * @test
+     * @testdox To understand reality, we must compare our expectations against reality
      */
-    public function testAssertEquality()
+    public function checksHowMuchIsOnePlusOneUsingVariables()
     {
-        $expected_value = __; // Replace __ with your answer
-        $actual_value = 1 + 1;
+        $expectedValue = __;
+        $actualValue = 1 + 1;
 
-        $this->assertTrue($expected_value == $actual_value);
+        $this->assertTrue($expectedValue == $actualValue);
     }
 
     /**
-     * @testdox Some ways of asserting equality are better than others.
+     * @test
+     * @testdox Some ways of asserting equality are better than others
      */
-    public function testABetterWayToAssertEquality()
+    public function checksHowMuchIsOnePlusOneUsingVariablesAndAssertEquals()
     {
-        $expected_value = __; // Replace __ with your answer
-        $actual_value = 1 + 1;
+        $expectedValue = __;
+        $actualValue = 1 + 1;
 
-        $this->assertEquals($expected_value, $actual_value);
+        $this->assertEquals($expectedValue, $actualValue);
     }
 
     /**
-     * @testdox Sometimes we need to know the variable type.
+     * @test
+     * @testdox Sometimes we need to know the variable type
      */
-    public function testSometimesWeNeedToKnowTheVariableType()
+    public function checksTheTypeOfAString()
     {
         $this->assertEquals(__, gettype("What am I"));
     }
 
     /**
-     * @testdox Sometimes we need to know the class type.
+     * @test
+     * @testdox Sometimes we need to know the class type
      */
-    public function testSometimesWeNeedToKnowTheClassType()
+    public function checksTheTypeOfAClass()
     {
         // See bottom of this file for class definition
         $object = new Enlightenment();
```

Comment:
The class you have defined at the end of this script extract it to a separate class